docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>Please change</p><p> </p><p> </p><blockquote>If a dataset has similar <strong>more than one discords</strong>, we should know how those discords are calculated in Matrix Profile.  </blockquote><p> </p><p>to:</p><p> </p><p>If a dataset has two similar subsequences that are far from the rest of subsequences, they cannot be easily discovered by matrixprofile.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>